pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -48,6 +49,14 @@ export class FileAutoComplete extends React.Component {
         this.filterFiles = this.filterFiles.bind(this);
         this.showAllOptions = this.showAllOptions.bind(this);
         this.selectItem = this.selectItem.bind(this);
+        this.debouncedChange = debounce(300, (value) => {
+            this.pendingCallback = false;
+            if (!this.updateIfDirectoryChanged(value)) {
+                var stateUpdate = this.filterFiles(value);
```

Comment:
Please no `var` in arrow functions, that has broken scoping. Use `let` or `const`.